LiskHQ/lisk
# Peers insert failed due smallint out of range - Closes #2496

### What was the problem?

In table peers column wsPort was of type small int with a max possible value of 32767 this caused peers.cleanup() to fail when the configured port for wsPort was set to a bigger value

### How did I fix it?

- Wrote a migration for changing the type of peers.wsPort to int
- Added a test to avoid someone accidentally change the schema back to small int

### How to test it?

- edit your peers wsPort configuration and set it to a number bigger than 32767
- run `node app -n testnet`
- ctrl + c (for stopping execution)
- there should not be errors in the console or logs about invalid ranges

### Review checklist

* The PR resolves #2496 
* All new code is covered with unit tests
* All new code was formatted with Prettier
* Linting passes
* Tests pass
* Commit messages follow the [commit guidelines](CONTRIBUTING.md#git-commit-messages)
* Documentation has been added/updated
